fr.esrf.tango.generator.xtend/src/fr/esrf/tango/pogo/generator/python/PythonTypeDefinitions.java

```diff
@@ -115,19 +115,19 @@ public static String pythonPropType (PropType propType) {
 	 */
 	public static String pythonPropTypeHL (PropType propType) {
 		
-		if (propType instanceof BooleanType)	   return "'bool'";
-		if (propType instanceof ShortType)  	   return "'int16'";
-		if (propType instanceof IntType)           return "'int32'";
-		if (propType instanceof UShortType)        return "'uint16'";
-		if (propType instanceof UIntType)      	   return "'uint32'";
-		if (propType instanceof FloatType)         return "'float'";
-		if (propType instanceof DoubleType)        return "'double'";
-		if (propType instanceof StringType)        return "'str'";
-		if (propType instanceof ShortVectorType)   return "('int16',)";
-		if (propType instanceof IntVectorType)     return "('int',)";
-		if (propType instanceof FloatVectorType)   return "('float',)";
-		if (propType instanceof DoubleVectorType)  return "('double',)";
-		if (propType instanceof StringVectorType)  return "('str',)";
+		if (propType instanceof BooleanType)	   return "'DevBoolean'";
+		if (propType instanceof ShortType)  	   return "'DevShort'";
+		if (propType instanceof IntType)           return "'DevLong'";
+		if (propType instanceof UShortType)        return "'DevUShort'";
+		if (propType instanceof UIntType)      	   return "'DevULong'";
+		if (propType instanceof FloatType)         return "'DevFloat'";
+		if (propType instanceof DoubleType)        return "'DevDouble'";
+		if (propType instanceof StringType)        return "'DevString'";
+		if (propType instanceof ShortVectorType)   return "'vector<DevShort>'";
```

Comment:
I don't think PyTango will map this to the correct Python type.  This looks like C++ code.

Reply:
Okay got the point. Please let me know, if I shall retain the previous data type definitions.

Reply:
Yes, go back to TANGO types, so this would be `DevVarShortArray`

Reply:
Okay. Done with commit
https://github.com/ska-telescope/pogo/commit/86cf5a9cbec4846e41c8932cfa080f095dde1e9c